server/src/logic/utils/model.utils.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiError, ERRORS } from "./../../models/index";
+
+export class ModelUtils {
```

Comment:
Hmm, okay. But what if you change modelutils to modelmiddleware? :-) we have all middleware functions inside middleware classes.